oracledb-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,1440 @@
+  collapsed: false,
+  title: 'Tablespace',
+  type: 'row',
+};
+
+local tablespaceSizePanel = {
+  datasource: promDatasource,
+  description: 'Shows the size overtime for the tablespace.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'bytes',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+  targets: [
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_bytes{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      legendFormat: '{{instance}} - {{tablespace}} - used',
+      range: true,
+      refId: 'Used Bytes',
+    },
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_free{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      hide: false,
+      legendFormat: '{{instance}} - {{tablespace}} - free',
+      range: true,
+      refId: 'Free',
+    },
+    {
+      datasource: promDatasource,
+      editorMode: 'code',
+      expr: 'oracledb_tablespace_max_bytes{instance=~"$instance",job=~"$job",tablespace=~"$tablespace"}',
+      hide: false,
+      legendFormat: '{{instance}} - {{tablespace}} - max',
+      range: true,
+      refId: 'Max',
+    },
+  ],
+  title: 'Tablespace Size',
+  type: 'timeseries',
+};
+
+{
+  grafanaDashboards+:: {
+    'oracledb-overview.json':
+      dashboard.new(
+        'OracleDB Overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        editable=true,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        graphTooltip='shared_crosshair',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        title='Other OracleDB dashboards',
```

Comment:
As there are no other OracleDB dashboards included in the mixin, this can be removed for now